src/makefile

```diff
@@ -99,8 +105,8 @@ GIT_VERSION := $(shell git describe --long --dirty --all --always | sed -e's/hea
 ###################################################################
 ###################################################################
 ifeq ($(FC), gfortran)
-	COMP=-fopenmp -lgomp -O3 -ffree-line-length-none -ftree-vectorize -fimplicit-none -funroll-loops -march=native \
-		-fno-protect-parens -ffast-math #-flto #
+	COMP=-fopenmp -lgomp -O3 -ffree-line-length-none -ftree-vectorize -fimplicit-none -funroll-loops  \
+		-fno-protect-parens -ffast-math #-flto #-march=native
```

Comment:
It does on my imac with gfortran 5. I didn't much care for my makefile changes so I rolled them back.